docs/configuration-file/comprehensive-example.md

```diff
@@ -0,0 +1,71 @@
+### Comprehensive Example
```

Comment:
I appreciate that we can't make this pull in the file automatically (without the macro hacks you mentioned), but I think I would still at least like to generate this file via a make rule. So maybe the `make docs` will build this before serving so we have a (better) chance of this not getting out-of-sync.